tsdb/db.go

```diff
@@ -549,6 +549,7 @@ func validateOpts(opts *Options, rngs []int64) (*Options, []int64) {
 	return opts, rngs
 }
 
+// TODO(bwplotka): Detect .tmp and .tmp-to-delete dirs and remove on start.
```

Comment:
Why a todo? The PR is small enough to add here. Then it will be within the same context why these tmp might folders exist anyway.